cvmfs/cvmfs_config

```diff
@@ -254,13 +257,14 @@ cvmfs_chksetup() {
     fi
   fi
 
+  AUTOMASTER=$(find /etc -name "auto.master" 2>/dev/null | head -1)
```

Comment:
I'd still prefer to use the hard coded paths.  I agree that `find` adds flexibility but I also find it a bit dangerous.  You could imagine funny setups where /etc or some sub path is on a large network volume.  Also, by failing we get to know if a distribution changes the location of auto.master in /etc, which is a good hint for us to take a second look whether something else changed, too.